tests/integration/test_vulnerability_detector/test_scan_results/data/test_cases/cases_duplicate_feeds.yaml

```diff
@@ -0,0 +1,61 @@
+- name: 'RedHat 8'
+  description: 'Insert RHEL 8 OVAL and JSON feed from local path'
+  configuration_parameters:
+    PROVIDER: 'redhat'
+    OS: '8'
+    OS_PATH: CUSTOM_REDHAT_OVAL_FEED_PATH
+    PATH: CUSTOM_REDHAT_JSON_FEED_PATH
+  metadata:
+    provider_name: 'Red Hat Enterprise Linux 8'
+    provider_json_name: 'JSON Red Hat Enterprise Linux'
+
+- name: 'Debian Buster'
+  description: 'Insert Debian Buster OVAL and JSON feed from local path'
+  configuration_parameters:
+    PROVIDER: 'debian'
+    OS: 'Buster'
+    OS_PATH: CUSTOM_DEBIAN_OVAL_FEED_PATH
+    PATH: CUSTOM_DEBIAN_JSON_FEED_PATH
+  metadata:
+    provider_name: 'Debian Buster'
+    provider_json_name: ''
+
+- name: 'Ubuntu Bionic'
+  description: 'Ubuntu Bionic provider'
+  configuration_parameters:
+    PROVIDER: 'canonical'
+    OS: 'bionic'
+    OS_PATH: CUSTOM_CANONICAL_OVAL_FEED_PATH
+  metadata:
+    provider_name: 'Ubuntu Bionic'
+    provider_json_name: ''
+
+- name: 'Amazon Linux 1'
+  description: 'Amazon Linux provider'
+  configuration_parameters:
+    PROVIDER: 'alas'
+    OS: 'amazon-linux'
+    OS_PATH: CUSTOM_ALAS_JSON_FEED_PATH
+  metadata:
+    provider_name: 'Amazon Linux 1'
+    provider_json_name: ''
+
+- name: 'Arch Linux'
+  description: 'Arch Linux provider'
+  configuration_parameters:
+    PROVIDER: 'arch'
+    OS: ''
+    PATH: CUSTOM_ARCHLINUX_JSON_FEED_PATH
+  metadata:
+    provider_name: 'Arch Linux'
+    provider_json_name: ''
+
+- name: 'Microsoft Security Update'
```

Comment:
```suggestion
- name: 'MSU'
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2575/commits/73dd596fd88d841d166213eb00ccae04e87dc800